pex/pep_425.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import
+
+import itertools
+
+from pex.orderedset import OrderedSet
+from pex.third_party.packaging.tags import Tag, parse_tag
+from pex.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Iterable, Iterator, List, Tuple
+
+    import attr  # vendor:skip
+else:
+    from pex.third_party import attr
+
+
+def _prepare_tags(tags):
+    # type: (Iterable[Tag]) -> Tuple[Tag, ...]
+    return tags if isinstance(tags, tuple) else tuple(OrderedSet(tags))
+
+
+@attr.s(frozen=True)
+class CompatibilityTags(object):
+    """An ordered set of PEP-425 compatibility tags.
+
+    See: https://www.python.org/dev/peps/pep-0425/#use
+    """
+
+    @classmethod
+    def from_stings(cls, tags):
```

Comment:
```suggestion
    def from_strings(cls, tags):
```

Reply:
I'll fix this in #1609 since that will need another PR burn for a doc change.

Reply:
Fixed over in #1609 here: https://github.com/pantsbuild/pex/pull/1609/commits/89bcd0fcab0d1a425be3fef85264a0b1cbbd49a1